xbmc/addons/kodi-dev-kit/include/kodi/addon-instance/pvr/Timers.h

```diff
@@ -464,6 +468,35 @@ class PVRTimer : public CStructHdl<PVRTimer, PVR_TIMER>
 
   /// @brief To get with @ref SetSeriesLink changed values.
   std::string GetSeriesLink() const { return m_cStructure->strSeriesLink; }
+
+  /// @brief **optional**\n
+  /// Parental age rating
+  void SetParentalRating(int parentalRating) { m_cStructure->iParentalRating = parentalRating; }
+
+  /// @brief To get with @ref SetParentalRating changed values.
+  int GetParentalRating() const { return m_cStructure->iParentalRating; }
+
+  /// @brief **optional**\n
+  /// Parental rating code.
+  void SetParentalRatingCode(const std::string& parentalRatingCode)
+  {
+    strncpy(m_cStructure->strParentalRatingCode, parentalRatingCode.c_str(),
+            sizeof(m_cStructure->strParentalRatingCode) - 1);
+  }
+
+  /// @brief To get with @ref SetParentalRatingCode.
+  std::string GetParentalRatingCode() const { return m_cStructure->strParentalRatingCode; }
```

Comment:
Commit message only talks about introduction of parental rating icon. Actually this commit contains much more changes, like introduction of more functionality for timers. The latter should be a separate commit of this PR.